c/test/CMakeLists.txt

```diff
@@ -7,7 +7,11 @@ else (CMAKE_CROSSCOMPILING)
     message(STATUS "Skipping unit tests, Check library not found!")
   else (NOT CHECK_FOUND)
 
-    set(TEST_LIBS ${TEST_LIBS} ${CHECK_LIBRARIES} pthread sbp m)
+    if(PLATFORM_IMPLEMENTATION_HAS_PTHREAD)
```

Comment:
Where is this defined?

Reply:
If this is a Swift specific option that we're enabling in an NDK build (since Bionic has pthread built-in), can we name this something like `LIBSBP_BUILTIN_PTHREADS`.

Reply:
```suggestion
    if(LIBSBP_BUILTIN_PTHREADS)
```